types/react-sortable-tree/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-sortable-tree 0.3
+// Type definitions for react-sortable-tree 2.7
```

Comment:
I do not think the version should be changed. The versions will not match in the longterm. There can be multiple updates to TS for just one version of the sortable tree and also when doing changes to sortable no one will come here to bump the version.
And this is not a breaking change, it is a bug fix.
If I remember it correctly some bot will bump the minor version with release.

Reply:
I see, I'll revert the version changes